org.parosproxy.paros.db.DatabaseException: java.sql.SQLException: Can not issue executeUpdate() or executeLargeUpdate() for SELECTs

I have just tried to replace the default HSQL with MYSQL on my windows 10 machine. However, I have the following error in the terminal after making the necessary changes in the file "mysql.schema".
`12549 [AWT-EventQueue-0] ERROR org.parosproxy.paros.model.Session  - java.sql.SQLException: Can not issue executeUpdate() or executeLargeUpdate() for SELECTs
org.parosproxy.paros.db.DatabaseException: java.sql.SQLException: Can not issue executeUpdate() or executeLargeUpdate() for SELECTs
        at org.zaproxy.zap.db.sql.SqlTableContext.deleteAllDataForContextAndType(Unknown Source)
        at org.zaproxy.zap.db.sql.SqlTableContext.setData(Unknown Source)
        at org.parosproxy.paros.model.Session.setContextData(Unknown Source)
        at org.parosproxy.paros.model.Session.setContextData(Unknown Source)
        at org.parosproxy.paros.model.Session.saveContext(Unknown Source)
        at org.parosproxy.paros.model.Model.newSession(Unknown Source)
        at org.parosproxy.paros.control.Control.createNewSession(Unknown Source)
        at org.parosproxy.paros.control.Control.newSession(Unknown Source)
        at org.parosproxy.paros.control.MenuFileControl.newSession(Unknown Source)
        at org.parosproxy.paros.control.MenuFileControl.newSession(Unknown Source)
        at org.zaproxy.zap.GuiBootstrap$3.run(Unknown Source)
        at java.awt.event.InvocationEvent.dispatch(Unknown Source)
        at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
        at java.awt.EventQueue.access$500(Unknown Source)
        at java.awt.EventQueue$3.run(Unknown Source)
        at java.awt.EventQueue$3.run(Unknown Source)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
        at java.awt.EventQueue.dispatchEvent(Unknown Source)
        at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
        at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
        at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
        at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
        at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
        at java.awt.EventDispatchThread.run(Unknown Source)
Caused by: java.sql.SQLException: Can not issue executeUpdate() or executeLargeUpdate() for SELECTs
        at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:964)
        at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:897)
        at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:886)
        at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860)
        at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2045)
        at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2009)
        at com.mysql.jdbc.PreparedStatement.executeLargeUpdate(PreparedStatement.java:5098)
        at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1994)
`

--- a/src/db/hsqldb.properties
+++ b/src/db/hsqldb.properties
@@ -53,8 +53,8 @@ context.ps.read						= SELECT * FROM CONTEXT_DATA WHERE DATAID = ?
 context.ps.insert					= INSERT INTO CONTEXT_DATA (CONTEXTID, TYPE, DATA) VALUES (?, ?, ?)
 context.ps.lastinsert				= CALL IDENTITY()
 context.ps.delete					= DELETE FROM CONTEXT_DATA WHERE  CONTEXTID = ? AND  TYPE = ? AND DATA = ?
-context.ps.alldataforcontext		= DELETE FROM CONTEXT_DATA WHERE  CONTEXTID = ?
-context.ps.alldataforcontexttype	= DELETE FROM CONTEXT_DATA WHERE  CONTEXTID = ? AND  TYPE = ?
+context.ps.deletealldataforcontext = DELETE FROM CONTEXT_DATA WHERE  CONTEXTID = ?
+context.ps.deletealldataforcontexttype = DELETE FROM CONTEXT_DATA WHERE  CONTEXTID = ? AND  TYPE = ?
 context.ps.alldata					= SELECT * FROM CONTEXT_DATA
 context.ps.alldataforcontext		= SELECT * FROM CONTEXT_DATA WHERE  CONTEXTID = ?
 context.ps.alldataforcontexttype	= SELECT * FROM CONTEXT_DATA WHERE  CONTEXTID = ? AND  TYPE = ?

--- a/src/db/mysql.properties
+++ b/src/db/mysql.properties
@@ -53,8 +53,8 @@ context.ps.read						= SELECT * FROM CONTEXT_DATA WHERE DATAID = ?
 context.ps.insert					= INSERT INTO CONTEXT_DATA (CONTEXTID, TYPE, DATA) VALUES (?, ?, ?)
 context.ps.lastinsert				= SELECT LAST_INSERT_ID()
 context.ps.delete					= DELETE FROM CONTEXT_DATA WHERE  CONTEXTID = ? AND  TYPE = ? AND DATA = ?
-context.ps.alldataforcontext		= DELETE FROM CONTEXT_DATA WHERE  CONTEXTID = ?
-context.ps.alldataforcontexttype	= DELETE FROM CONTEXT_DATA WHERE  CONTEXTID = ? AND  TYPE = ?
+context.ps.deletealldataforcontext = DELETE FROM CONTEXT_DATA WHERE  CONTEXTID = ?
+context.ps.deletealldataforcontexttype = DELETE FROM CONTEXT_DATA WHERE  CONTEXTID = ? AND  TYPE = ?
 context.ps.alldata					= SELECT * FROM CONTEXT_DATA
 context.ps.alldataforcontext		= SELECT * FROM CONTEXT_DATA WHERE  CONTEXTID = ?
 context.ps.alldataforcontexttype	= SELECT * FROM CONTEXT_DATA WHERE  CONTEXTID = ? AND  TYPE = ?

--- a/src/db/mysql.schema
+++ b/src/db/mysql.schema
@@ -55,10 +55,11 @@ DROP TABLE IF EXISTS `CONTEXT_DATA`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `CONTEXT_DATA` (
-  `DATAID` bigint(20) NOT NULL,
+  `DATAID` bigint(20) NOT NULL AUTO_INCREMENT,
   `CONTEXTID` int(11) NOT NULL,
   `TYPE` int(11) NOT NULL,
-  `DATA` varchar(8192) DEFAULT ''''
+  `DATA` varchar(8192) DEFAULT '''',
+  PRIMARY KEY (`DATAID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
@@ -181,9 +182,10 @@ DROP TABLE IF EXISTS `SESSION_URL`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `SESSION_URL` (
-  `URLID` bigint(20) NOT NULL,
+  `URLID` bigint(20) NOT NULL AUTO_INCREMENT,
   `TYPE` int(11) NOT NULL,
-  `URL` varchar(8192) DEFAULT ''''
+  `URL` varchar(8192) DEFAULT '''',
+  PRIMARY KEY (`URLID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 /*!40101 SET character_set_client = @saved_cs_client */;
 

--- a/src/org/zaproxy/zap/db/sql/SqlTableContext.java
+++ b/src/org/zaproxy/zap/db/sql/SqlTableContext.java
@@ -129,7 +129,7 @@ public class SqlTableContext extends SqlAbstractTable implements TableContext {
 	public synchronized void deleteAllDataForContextAndType(int contextId, int type) throws DatabaseException {
     	SqlPreparedStatementWrapper psDeleteAllDataForContextAndType = null;
     	try {
-        	psDeleteAllDataForContextAndType = DbSQL.getSingleton().getPreparedStatement("context.ps.alldataforcontexttype");
+        	psDeleteAllDataForContextAndType = DbSQL.getSingleton().getPreparedStatement("context.ps.deletealldataforcontexttype");
 			psDeleteAllDataForContextAndType.getPs().setInt(1, contextId);
 			psDeleteAllDataForContextAndType.getPs().setInt(2, type);
 			psDeleteAllDataForContextAndType.getPs().executeUpdate();
@@ -147,7 +147,7 @@ public class SqlTableContext extends SqlAbstractTable implements TableContext {
 	public synchronized void deleteAllDataForContext(int contextId) throws DatabaseException {
     	SqlPreparedStatementWrapper psDeleteAllDataForContext = null;
     	try {
-        	psDeleteAllDataForContext = DbSQL.getSingleton().getPreparedStatement("context.ps.alldataforcontext");
+        	psDeleteAllDataForContext = DbSQL.getSingleton().getPreparedStatement("context.ps.deletealldataforcontext");
 			psDeleteAllDataForContext.getPs().setInt(1, contextId);
 			psDeleteAllDataForContext.getPs().executeUpdate();
 		} catch (SQLException e) {

